From 45c0843ad361c482608e6469d3b4bd2b1b3561f3 Mon Sep 17 00:00:00 2001
From: Xu Bing <bing.xu@intel.com>
Date: Fri, 30 Aug 2024 10:05:45 +0800
Subject: [PATCH] Fix ATS test error on DefaultPermissionGrantPolicyTest

Add default permissions for package com.android.networkstack and
com.android.networkstack.tethering because test case will check
them.

Tracked-On: OAM-122852
Signed-off-by: Xu Bing <bing.xu@intel.com>
---
 data/etc/privapp-permissions-platform.xml | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/data/etc/privapp-permissions-platform.xml b/data/etc/privapp-permissions-platform.xml
index b6771b7aa855..e685fe8e3c0e 100644
--- a/data/etc/privapp-permissions-platform.xml
+++ b/data/etc/privapp-permissions-platform.xml
@@ -588,4 +588,19 @@ applications that come with the platform
         <permission name="android.permission.PACKAGE_USAGE_STATS"/>
         <permission name="android.permission.GET_TASKS"/>
     </privapp-permissions>
+
+    <privapp-permissions package="com.android.networkstack">
+        <permission name="android.permission.ACCESS_FINE_LOCATION"/>
+        <permission name="android.permission.ACCESS_COARSE_LOCATION"/>
+    </privapp-permissions>
+
+    <privapp-permissions package="com.android.networkstack.tethering">
+        <permission name="android.permission.POST_NOTIFICATIONS"/>
+        <permission name="android.permission.BLUETOOTH_CONNECT"/>
+    </privapp-permissions>
+
+    <privapp-permissions package="com.intel.multicamera">
+        <permission name="android.permission.CAMERA"/>
+        <permission name="android.permission.RECORD_AUDIO"/>
+    </privapp-permissions>
 </permissions>
-- 
2.34.1

